tests/unit/test_lsa_to_madx.py

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
maybe we can also do a version of the test without the deferred expressions

Reply:
It doesn't test much anyway. And I don't like these tests, as you need to change the file it compares to as soon as you do any change in the code. A proper test would be running the resulting file in madx/cpymad and checking that the variables change as expected, but I don't have the time to implement this.